.travis.yml

```diff
@@ -12,15 +12,19 @@ stages:
   - name: unsupported
     if: branch = master AND type = push
 
+.disable_global: &disable_global
```

Comment:
Remove those changes which are unrelated to your pr.

Reply:
You didn't do what I said, and you are adding more irrelevant changes :(

I am pretty sure you are not using git command correctly

Fix the problem first. Next time before you push, use `gitk` or `gitg` tools, or use `git diff head~1` to check the changes within your commit.